### DISCUSSION TITLE
Risk Tracker: What number should another microCOVID user tell me so I can log an activity with them?

### DISCUSSION THREAD

**Discussion by nickybecky1** [microCOVID/microCOVID#1109] [Q&A] [answered]
Hi! When I'm adding an internal person who also used microcovid, which number do I use for "Current microCOVID Person Risk" on the Internal_Person spreadsheet? Should it be risk to self, pod, or others? 

Thanks!

**Comment by blanchardjeremy** [comment, marked as answer]
Hi @nickybecky1

If the person is using microCOVID and you want to enter their exact current microCOVID score:

1. Go to **Activity Log** sheet
1. Enter the activity parameters
1. Under "risk profile" just type a number instead of using the dropdown.

If you don't know their exact score, but you know they keep a 1% budget, you can select "1% risk per year adherent (using microCOVID)" as their risk profile.

Does that help?

--- 
Update:

To find the number that the other person using microCOVID should share with you, Look for the number in purple on the pod overview sheet. (This number changes each day as their risk "decays" as previous exposures become less risky over time.)

![image](https://user-images.githubusercontent.com/32290/132414002-5a1ea9d2-73a6-47e7-80d7-6c38b09e1944.png)

**Reply by nickybecky1** [reply]
I guess I'm just wondering which number from their spreadsheet they should tell me. Is it the risk to pod, risk to me, or risk to others number that I put into my spreadsheet to accurately count? There's a huge difference between the risk to pod and risk to others so I just want to make sure I'm using the right number. Thanks!

**Reply by blanchardjeremy** [reply]
(updated original answer)

**Comment by nickybecky1** [comment]
Great! Thanks!
